srcpkgs/x265/template

```diff
@@ -29,7 +30,10 @@ esac
 
 if [ "$build_option_assembly" ]; then
 	case "$XBPS_TARGET_MACHINE" in
-		x86_64*|i686*) hostmakedepends+=" nasm" ;;
+		x86_64*|i686*)
+			hostmakedepends+=" nasm"
+			configure_args+=" -DCMAKE_ASM_NASM_FLAGS=-w-macro-params-legacy"
```

Comment:
Can you add a comment for the flag? Is it because our nasm is too old/new or here to stay?

Reply:
can i just put this issue as comment? https://bitbucket.org/multicoreware/x265_git/issues/559/warnings-when-assembling-with-nasm-215

also i can remove it if you want, it produce too much warnings without this flag so makes it hard to read the logs.

Reply:
The comment is great, thanks.